src/components/BorrowerProfile/LoanProgress.vue

```diff
@@ -15,22 +15,34 @@
 			:bg-variant="'tertiary'"
 		/>
 		<figcaption class="tw-flex">
-			<div v-if="loanStatus === 'funded' || loanStatus === 'raised'">
-				<p class="tw-text-h3 tw-m-0" data-testid="bp-summary-amount-to-go">
-					This loan is fully funded!
-				</p>
-				<div class="md:tw-flex tw-gap-2">
-					<p class="tw-text-h4 tw-text-secondary tw-block">
-						100% funded
-					</p>
-					<p class="tw-text-h4 tw-text-action tw-block">
-						<router-link
-							:to="`/lend-classic/${$route.params.id}?minimal=false`"
-							v-kv-track-event="['Lending', 'full-borrower-profile-exit-link']"
-						>
-							View the full borrower profile
-						</router-link>
+			<div
+				v-if="loanStatus === 'funded' || loanStatus === 'raised'"
+				class="tw-w-full"
```

Comment:
Some minor changes in the styles, does this component still look good on the borrower profile page?

Reply:
Yeah, works as expected.